src/src/config.cpp

```diff
@@ -134,17 +134,20 @@ TxConfig::SetDefaults()
     SetSSID("");
     SetPassword("");
     for (int i=0 ; i<64 ; i++) {
-        SetRate(i, modParams->index);
-        SetTlm(i, modParams->TLMinterval);
-        SetPower(i, DefaultPowerEnum);
-        SetSwitchMode(i, 1);
-        SetModelMatch(i, false);
+        SetModelId(i);
+        SetRate(modParams->index);
+        SetTlm(modParams->TLMinterval);
+        SetPower(DefaultPowerEnum);
+        SetSwitchMode(1);
+        SetModelMatch(false);
     }
     SetVtxBand(0);
     SetVtxChannel(0);
     SetVtxPower(0);
     SetVtxPitmode(0);
     Commit();
+
+    SetModelId(0);
```

Comment:
I think you should save the current modelId before the iteration and restore it afterward because if the modelid is not 0 then it's going to be wrong here.

Reply:
I thought about that, but this is "SetDefault" so I thought it should set everything to default and this is only called on startup before communication even starts so I figured I'd save a few bytes of flash by forcing it to 0. If you think we need it, let me know and I'll update to save/restore the modelid on reset.

Reply:
Yes I was just about to retract my comment because of that 😄